App/Models/TextileTypes.ts

```diff
@@ -22,6 +22,13 @@ export interface Devices {
   items: Device[]
 }
 
+export interface Notification {
```

Comment:
I originally made the `TextileTypes` file to keep track of the model types that are used with `textile-go`... Let's make a point of cleaning that up, makes it easier to deal with the framework update process.

Reply:
cool. pretty sure this guy will live here regardless, it's the UI-based reshaping of notifications. making it so i can easily style the feed list item by type of notification, stuff that go isn't going to care about